src/actions/auth.js

```diff
@@ -2,6 +2,8 @@ import firebase from "../firebase";
 import { handleError } from "../utils";
 import { fetchProfile } from "./profile";
 
+export const USER_NOT_LOGGED = "USER_NOT_LOGGED";
```

Comment:
A nit: `USER_NOT_SIGNED_IN` to be consistent with the other actions?

Reply:
Ok! If done better and now all the actions are using the async action factories.
I was thinking in something better than the first state being `loading: null` but for now looks good :)